lib/matplotlib/tests/test_dates.py

```diff
@@ -1081,18 +1081,6 @@ def test_change_interval_multiples():
     assert ax.get_xticklabels()[1].get_text() == 'Feb 01 2020'
 
 
-def test_epoch2num():
```

Comment:
I think we leave tests for deprecated features in place and run them `with _api.suppress_matplotlib_deprecation_warning():`.